redbadger/count-dracula
# Scaffold a web app

Closes #2 

## Technical acceptance criteria
- [x]  Create a react app with redux

## How to test 
- On this branch, from the root run the following:
```sh
yarn workspace web start
```
- Open the browser and check [http://localhost:3000](http://localhost:3000)
- 🚀
